lib/node_modules/@stdlib/stats/base/dists/bernoulli/mean/benchmark/benchmark.js

```diff
@@ -40,7 +40,7 @@ bench( pkg, function benchmark( b ) {
 	len = 100;
 	p = new Float64Array( len );
 	for ( i = 0; i < len; i++ ) {
-		p[ i ] = ( randu()*1.0 ) + EPS;
+		p[ i ] = uniform( EPS, EPS + 1.0 );
```

Comment:
```suggestion
		p[ i ] = uniform( EPS, 1.0 );
```

`p` is a success probability, so you never want it to exceed `1`. The previous worked, as `randu()` is always `< 1`.